js/modal.js

```diff
@@ -259,11 +260,12 @@
 
   Modal.prototype.setScrollbar = function () {
     var bodyPad = parseInt((this.$body.css('padding-right') || 0), 10)
+    this.originalBodyPad = this.$body.attr('style') ? bodyPad : null
```

Comment:
Much better - I keep forgetting that `Element.style` does not include computed styles.